about.html

```diff
@@ -0,0 +1,134 @@
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <div id="campaign">
+            <span><i class="fas fa-microphone"></i>  Campaign</span>
+          </div>
+        </div>
+      </nav>
+        <span>
+          <i class="fas fa-bars fa-2x" id="menu"></i>
+        </span>
+        <span id="home">
+          <a href="./index.html" id="home-link"><i class="fas fa-home fa-1x"></i></a>
+        </span>
+        <span id="salutation">Hello Worshippers! Welcome.</span>
+        <h1>MELLISA
+          <span class="h1-span">LIVE ALBUM <br>LAUNCH</span>
+          <span class="h1-span">2022</span>
+        </h1>
+        <div class="red-line" id="hidden-line"></div>
+        <p id="main-p">Mellisa Live Album Launch is an ongoing event starting in Zimbabwe and spreading throughout the world. By words uttered over her, you are guaranteed to experience deep worship by merely your presence.</p>
+        <p id="contact">Please contact us by email for any further questions about <span><i class="fas fa-microphone"></i></span>Mellisa Album Launch 2022</p>
+        <span id="email">info@mellisamusic.org</span>
+    </header>
+    <main>
+      <section id="logo-section">
+        <h2>Mellisa Live Album<br> Launch 2022</h2>
+        <div class="red-line"></div>
+        <p>The logo of <span><i class="fas fa-microphone"></i></span>Mellisa Album Launch 2022 was decided through the logo competition from
+          the 6th of May to the 12th of July.</p>
+        <div class="logo-container">
+          <span>
+            <i class="fas fa-microphone fa-4x"></i>
+          </span>
+          <div>
+            <span>Mellisa</span>
+            <span>Live Album Launch 2022</span>
+          </div>
+        </div>
+      </section>
+      <section id="pastevents">
+        <h2>Past Live Events By Mellisa</h2>
+        <div class="red-line"></div>
+        <p>Have a glance at the past worship experiences captured live and anticipate what you will experience.</p>
+        <div id="red-container">
+          <div class="containers" id="container-1">
+            <div class="red-div">
+              <h4>2018</h4>
+              <span><i class="fas fa-microphone"></i>Woship Night Concert</span>
+            </div>
+          </div>
+          <div class="containers" id="container-2">
+            <div class="red-div">
+              <h4>2019</h4>
+              <span><i class="fas fa-microphone"></i>UFIC Mutare City</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="partners">
+        <h2>Partners</h2>
+        <div class="division-line"></div>
+        <div id="partners-wrap">
+          <span>Logitap</span>
+          <span>Magic Pixels</span>
+          <span>Park Cafeteria</span>
+          <span>Mura Meats</span>
+          <span>Hillside Kitchen</span>
+          <span>Melz Cakes</span>
+        </div>
+      </section>
+    </main>
+    <footer>
```

Comment:
- [ ] [Optional] The footer should be the same for the ```Home page``` and the ```About page```, the only difference is the color, please reduce the padding and consider moving the logo to the left just like in the home page

- it should be something like this:
![Screenshot_20220519_214632](https://user-images.githubusercontent.com/25227022/169401749-49442d1b-c4c1-449d-924a-5aa51ea3a36e.png)

Reply:
Thank you, implemented the suggestion.